rlwrap.yaml

```diff
@@ -0,0 +1,10 @@
+name: rlwrap
+sources:
+  - type: archive
+    url: https://github.com/hanslub42/rlwrap/archive/v0.43.tar.gz
+    sha256: 29e5a850fbe4753f353b0734e46ec0da043621bdcf7b52a89b77517f3941aade
+  - type: shell
+    commands:
+      - autoreconf -vfi;
+post-install:
+  - install -p -D -m 0644 "COPYING" -t "${FLATPAK_DEST}/share/licenses/rlwrap/";
```

Comment:
It doesn't matter, it's part of Sources extension.

Reply:
@barthalion 

> It doesn't matter, it's part of Sources extension.

Will you correct it here or you don't care as usual?
https://github.com/flathub/com.github.wwmm.pulseeffects/blob/3ca9162e35e63390ec3cbed2cb050e4e9e40dedc/com.github.wwmm.pulseeffects.yml#L42
https://github.com/flathub/com.github.wwmm.pulseeffects/blob/3ca9162e35e63390ec3cbed2cb050e4e9e40dedc/com.github.wwmm.pulseeffects.yml#L56
https://github.com/flathub/com.github.wwmm.pulseeffects/blob/3ca9162e35e63390ec3cbed2cb050e4e9e40dedc/com.github.wwmm.pulseeffects.yml#L66
https://github.com/flathub/com.github.wwmm.pulseeffects/blob/3ca9162e35e63390ec3cbed2cb050e4e9e40dedc/com.github.wwmm.pulseeffects.yml#L80
https://github.com/flathub/com.github.wwmm.pulseeffects/blob/3ca9162e35e63390ec3cbed2cb050e4e9e40dedc/com.github.wwmm.pulseeffects.yml#L92
https://github.com/flathub/com.github.wwmm.pulseeffects/blob/3ca9162e35e63390ec3cbed2cb050e4e9e40dedc/com.github.wwmm.pulseeffects.yml#L106
https://github.com/flathub/com.github.wwmm.pulseeffects/blob/3ca9162e35e63390ec3cbed2cb050e4e9e40dedc/com.github.wwmm.pulseeffects.yml#L118
https://github.com/flathub/com.github.wwmm.pulseeffects/blob/3ca9162e35e63390ec3cbed2cb050e4e9e40dedc/com.github.wwmm.pulseeffects.yml#L130
https://github.com/flathub/com.github.wwmm.pulseeffects/blob/3ca9162e35e63390ec3cbed2cb050e4e9e40dedc/com.github.wwmm.pulseeffects.yml#L155
https://github.com/flathub/com.github.wwmm.pulseeffects/blob/3ca9162e35e63390ec3cbed2cb050e4e9e40dedc/com.github.wwmm.pulseeffects.yml#L205
https://github.com/flathub/com.github.wwmm.pulseeffects/blob/3ca9162e35e63390ec3cbed2cb050e4e9e40dedc/com.github.wwmm.pulseeffects.yml#L216
https://github.com/flathub/com.github.wwmm.pulseeffects/blob/3ca9162e35e63390ec3cbed2cb050e4e9e40dedc/com.github.wwmm.pulseeffects.yml#L254
https://github.com/flathub/com.github.wwmm.pulseeffects/blob/3ca9162e35e63390ec3cbed2cb050e4e9e40dedc/com.github.wwmm.pulseeffects.yml#L270
https://github.com/flathub/com.github.wwmm.pulseeffects/blob/3ca9162e35e63390ec3cbed2cb050e4e9e40dedc/com.github.wwmm.pulseeffects.yml#L286
https://github.com/flathub/com.github.wwmm.pulseeffects/blob/3ca9162e35e63390ec3cbed2cb050e4e9e40dedc/com.github.wwmm.pulseeffects.yml#L304
https://github.com/flathub/com.github.wwmm.pulseeffects/blob/3ca9162e35e63390ec3cbed2cb050e4e9e40dedc/com.github.wwmm.pulseeffects.yml#L320
https://github.com/flathub/com.github.wwmm.pulseeffects/blob/3ca9162e35e63390ec3cbed2cb050e4e9e40dedc/com.github.wwmm.pulseeffects.yml#L336
https://github.com/flathub/com.github.wwmm.pulseeffects/blob/3ca9162e35e63390ec3cbed2cb050e4e9e40dedc/com.github.wwmm.pulseeffects.yml#L344
https://github.com/flathub/com.github.wwmm.pulseeffects/blob/3ca9162e35e63390ec3cbed2cb050e4e9e40dedc/com.github.wwmm.pulseeffects.yml#L367
https://github.com/flathub/com.github.wwmm.pulseeffects/blob/3ca9162e35e63390ec3cbed2cb050e4e9e40dedc/com.github.wwmm.pulseeffects.yml#L396
https://github.com/flathub/com.github.wwmm.pulseeffects/blob/3ca9162e35e63390ec3cbed2cb050e4e9e40dedc/com.github.wwmm.pulseeffects.yml#L416
https://github.com/flathub/com.github.wwmm.pulseeffects/blob/3ca9162e35e63390ec3cbed2cb050e4e9e40dedc/com.github.wwmm.pulseeffects.yml#L426
https://github.com/flathub/com.github.wwmm.pulseeffects/blob/3ca9162e35e63390ec3cbed2cb050e4e9e40dedc/com.github.wwmm.pulseeffects.yml#L439

See also:
https://github.com/flathub/flathub/pull/72#commits-pushed-f8778fe

Reply:
More examples:
https://github.com/flathub/com.github.JannikHv.Gydl/blob/7ff59619213c1c9ce59921bbea2ea762b8fa1dc6/com.github.JannikHv.Gydl.json#L40
https://github.com/flathub/com.github.JannikHv.Gydl/blob/7ff59619213c1c9ce59921bbea2ea762b8fa1dc6/com.github.JannikHv.Gydl.json#L71
https://github.com/flathub/com.github.wwmm.pulseeffects.plugin.lsp-plugins/blob/440891a325fe8500b1c405feb545e05a2b2d5c10/com.github.wwmm.pulseeffects.plugin.lsp-plugins.yml#L29
https://github.com/flathub/com.github.wwmm.pulseeffects.plugin.lsp-plugins/blob/440891a325fe8500b1c405feb545e05a2b2d5c10/com.github.wwmm.pulseeffects.plugin.lsp-plugins.yml#L55
https://github.com/flathub/com.makemkv.MakeMKV/blob/53b4996c627e36bc8c503f2261197be995ba6b61/com.makemkv.MakeMKV.json#L120
https://github.com/flathub/com.makemkv.MakeMKV/blob/53b4996c627e36bc8c503f2261197be995ba6b61/com.makemkv.MakeMKV.json#L143
https://github.com/flathub/com.snes9x.Snes9x/blob/7f86af79009ed40bfa7e653441380f9104ebaf43/com.snes9x.Snes9x.json#L28
https://github.com/flathub/de.manuel_kehl.go-for-it/blob/646aeb5016bb0d13d609d39e6f8f879e00c26e40/de.manuel_kehl.go-for-it.yml#L42
https://github.com/flathub/io.ark.Desktop/blob/56ab9997e33d8c1cda0358a58e01284ed29f2ab3/io.ark.Desktop.json#L34
https://github.com/flathub/org.keepassxc.KeePassXC/blob/99acb77c5246fedc7aa683c260b1ed595296dd08/org.keepassxc.KeePassXC.yml#L71-L74
https://github.com/flathub/org.keepassxc.KeePassXC/blob/99acb77c5246fedc7aa683c260b1ed595296dd08/org.keepassxc.KeePassXC.yml#L91
https://github.com/flathub/org.keepassxc.KeePassXC/blob/99acb77c5246fedc7aa683c260b1ed595296dd08/org.keepassxc.KeePassXC.yml#L104
https://github.com/flathub/org.keepassxc.KeePassXC/blob/99acb77c5246fedc7aa683c260b1ed595296dd08/org.keepassxc.KeePassXC.yml#L134
https://github.com/flathub/org.keepassxc.KeePassXC/blob/99acb77c5246fedc7aa683c260b1ed595296dd08/org.keepassxc.KeePassXC.yml#L146
https://github.com/flathub/org.keepassxc.KeePassXC/blob/99acb77c5246fedc7aa683c260b1ed595296dd08/org.keepassxc.KeePassXC.yml#L162

Reply:
Oh, it seems that this manifest was actually created by one of the Flathub maintainers (@TingPing):
https://github.com/flathub/com.snes9x.Snes9x/blob/c8a96f3f3f0a263f2ac83a8b5861eb3a08fee543/com.snes9x.Snes9x.json#L25

What's more, it looks like this package was also maintained by **you** and @nedrichards:
https://github.com/flathub/com.snes9x.Snes9x/commit/cecb76f8edf33368eca01a48bdfa5a205b3fdfc4
https://github.com/flathub/com.snes9x.Snes9x/commit/42864ab73f166cf397b660bc8e737742f41e5b57

Here we have another example:
https://github.com/flathub/io.ark.Desktop/blob/69e0aac3cd60a906132ea2033666ec733603609c/io.ark.Desktop.json#L34
This is also by @TingPing.

Do you know why the Flathub maintainers don't want this place to be seen as source of good practices?

Of course, I can fix it for wxMaxima, but I expect you to do the same for the rest of mentioned packages. :)

Reply:
You know how to submit pull requests.

Reply:
@barthalion 

> You know how to submit pull requests.

And I already know that you will ignore it, just like here:
https://github.com/flathub/org.gnome.gbrainy/pull/4

You know, I even considered yamling the manifest and changing runtime to the GNOME 3.30 (it wasn't possible before due to missing Mono extension for Freedesktop 18.08), but this is pointless, because it clearly doesn't matter to you at all.

Reply:
Done.

https://github.com/flathub/flathub/pull/801/commits/de93f9e911a070beb1913aade48a4b7f544a2eb9#diff-9211130e517ae4f676a29c779a3c6701L10